draft-ietf-quic-qpack.md

```diff
@@ -207,12 +207,35 @@ evicted.  Hence the encoder needs to track information about each compressed
 header block that references the dynamic table until that header block is
 acknowledged by the decoder.
 
+
+### Largest Known Received
+
+In order to identify which dynamic table entries can be safely used
+without a stream becoming blocked, the encoder tracks the absolute index of the
+decoder's Largest Known Received entry.
+
+When blocking references are permitted, the encoder uses header block
+acknowledgement to identify the Largest Known Received index, as described in
+{{header-acknowledgement}}.
+
+To acknowledge dynamic table entries which are not referenced by header blocks,
+for example because the encoder or the decoder have chosen not to risk blocked
+streams, the decoder sends a Table State Synchronize instruction (see
+{{table-state-synchronize}}).
+
+
 ### Blocked Dynamic Table Insertions {#blocked-insertion}
 
+The encoder considers a dynamic table entry to be blocking if its absolute index
+is larger than the Largest Known Received index or if not all references to the
+entry sent on a request or push stream have been acknowledged by the decoder
+using Header Block Acknowledgement instructions.  Note that a dynamic table
+entry that has never been referenced can still be blocking.
+
 An encoder MUST NOT insert an entry into the dynamic table (or duplicate an
-existing entry) if doing so would evict an entry with unacknowledged references.
-For header blocks that might rely on the newly added entry, the encoder can use
-a literal representation and maybe insert the entry later.
+existing entry) if doing so would evict a blocking entry.  For header blocks
+that would rely on the newly added entry, the encoder can instead use a literal
+representation and maybe insert the entry later.
```

Comment:
With the added language, this line feels more ambiguous about which "the entry" is being referenced.  (Of course, it was somewhat ambiguous before.)  Maybe just drop the "and maybe insert the entry later"?